.env.example

```diff
@@ -0,0 +1,50 @@
+# Base Path Opensift: /businesses/create Firebase: /
+VUE_APP_PATH=/businesses/create
+
+#vaults Shared
+VUE_APP_ADDRESS_COMPLETE_KEY=
+
+#vaults web-url
+VUE_APP_REGISTRY_HOME_URL="https://dev.bcros.ca/"
+VUE_APP_AUTH_WEB_URL="https://dev.bcregistry.ca/business/auth/"
+VUE_APP_BUSINESSES_URL="https://dev.bcregistry.ca/business/auth/"
+VUE_APP_DASHBOARD_URL="https://dev.bcregistry.ca/business/"
+VUE_APP_SITEMINDER_LOGOUT_URL="https://logontest7.gov.bc.ca/clp-cgi/logoff.cgi"
+
+#vaults API
+VUE_APP_AUTH_API_URL="https://auth-api-dev.apps.silver.devops.gov.bc.ca"
+VUE_APP_AUTH_API_VERSION="/api/v1"
+VUE_APP_LEGAL_API_URL="https://legal-api-dev.apps.silver.devops.gov.bc.ca"
+VUE_APP_LEGAL_API_VERSION_2="/api/v2"
+VUE_APP_NAICS_API_URL="https://legal-api-dev.apps.silver.devops.gov.bc.ca"
+VUE_APP_NAICS_API_VERSION_2="/api/v2"
+VUE_APP_STATUS_API_URL="https://status-api-dev.apps.silver.devops.gov.bc.ca"
+VUE_APP_STATUS_API_VERSION="/api/v1"
+VUE_APP_PAY_API_URL="https://pay-api-dev.apps.silver.devops.gov.bc.ca"
+VUE_APP_PAY_API_VERSION="/api/v1"
+VUE_APP_REGISTRIES_SEARCH_API_URL="https://bcregistry-dev.apigee.net/registry-search"
+VUE_APP_REGISTRIES_SEARCH_API_VERSION="/api/v1"
+VUE_APP_REGISTRIES_SEARCH_API_KEY=
+
+#vaults launchdarkly
+VUE_APP_BUSINESS_CREATE_LD_CLIENT_ID=
+
+#vaults keycloak
+VUE_APP_KEYCLOAK_AUTH_URL="https://dev.loginproxy.gov.bc.ca/auth"
+VUE_APP_KEYCLOAK_REALM="bcregistry"
+VUE_APP_KEYCLOAK_CLIENTID="entity-web"
+
+#vaults sentry
+VUE_APP_SENTRY_DSN=
+
```

Comment:
Missing "SENTRY_ENABLE" ... but I don't know if it's being used.

Reply:
Oh, is it a FF now?

Reply:
Yes, FF.